.env.template

```diff
@@ -16,7 +16,7 @@ TREASURY_SHARE=100 # treasuryShare, means 1%
 
 # Constructor parameters
 STORAGE_COST_L1=1500000000000000 # storage cost per blob - https://ethresear.ch/t/ethstorage-scaling-ethereum-storage-via-l2-and-da/14223/6#incentivization-for-storing-m-physical-replicas-1
-STORAGE_COST_L2=570000000000000 
+STORAGE_COST_L2=570000000000000000
```

Comment:
PREPAID_AMOUNT_L1=3145728000000000000000 # prepaidAmount - 50% * 2^39 / 131072 * 1500000 Gwei, it also means 3145 ETH for half of the shard
PREPAID_AMOUNT_L2=119537664000000000000000 # prepaidAmount - 50% * 2^39 / 131072 * 570000000 Gwei, it also means around 1,200,000 QKC for half of the shard

Because their units are GWei